ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -117,9 +143,35 @@ import UIKit
                                            paywallResultHandler: paywallResultHandler)
     }
 
+    @objc
+    public func presentPaywallIfNeeded(options: [String:Any],
```

Comment:
I'm wondering if we should deprecate all these functions and just use the new `presentPaywall` with options instead... Wdyt? We can also do that later though.

Reply:
yes, let's deprecate them, I hate them haha